CHANGELOG.rst

```diff
@@ -1,3 +1,7 @@
+**2.0.1 - 08/21/24**
+
+ - Use script to install dependencies in CI
+
```

Comment:
Does this actually need a patch release?

Reply:
Maybe not, but we want to make sure that th released version always matches main and releases are cheap